### PR TITLE
fix #17 Unicode characters in slugs

### LINKED ISSUE #17: Unicode characters in slugs
Bakery urlencodes URLs when creating folder names.
Which is indeed the URL asked by the browser to the HTTP server, but the HTTP server decodes the URL before searching the file in the system.

### PR DESCRIPTION
**Pull request by ckljohn**

### CODE REVIEW

File: tests/integration/test_models.py [hunk 1]
```diff
@@ -16,19 +16,28 @@ def test_site(site):
...
+    assert grandchildren[1].url == '/first/second/'
+
+    # Check if children of unicode-children return correct urls
+    grandchildren = children[3].get_children()
+    assert grandchildren.count() == 3
+
+    # TODO: @loicteixeira Fix unicode slugs in test?
+    # assert grandchildren[0].url == '/unicode-children/latin-capital-letter-i-with-diaeresis-Ï/'
+    # assert grandchildren[1].url == '/unicode-children/cyrillic-capital-letter-ya-Я/'
+    # assert grandchildren[2].url == '/unicode-children/cjk-fire-火/'
```
Comment (loicteixeira) [inline tests/integration/test_models.py:40]: Note to self @loicteixeira

**Comment by loicteixeira** [comment]
Thanks for your contribution @ckljohn 👍 

I've added a couple tests to make sure we don't break your fix in the future. Once I can get the second one working 100%, I'll merge this.

**Review by loicteixeira — APPROVED** [review]
I tested building the example site with Unicode characters in the slug and it worked.
However, the title didn't display right in the page body, but that's a separate issue (Edit: Adding `<meta charset="utf-8" />` to the base template fixed the issue).
Thank you for your contribution @ckljohn 👍

**Comment by ckljohn** [comment]
Thank you @loicteixeira for the tests and the rest of the work.

**Pull request merged**